frontend/src/components/Login/LoginDesktop.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { styled } from '@mui/material/styles';
+import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+import Paper from '@mui/material/Paper';
+import './login.css';
+
+const Item = styled(Paper)(({ theme }) => ({
+    backgroundColor: theme.palette.mode === 'dark' ? '#1A2027' : '#fff',
+    ...theme.typography.body2,
+    padding: theme.spacing(1),
+    textAlign: 'center',
+    color: theme.palette.text.secondary,
+  }));
+
+export default function LoginDesktop() {
+    return (
+        <div id="wrapper1">
+            <div id="loginField">
+                <h1>Account Log In</h1>
+                <form>
+                <TextField id="userIn" variant="outlined" placeholder='Username'></TextField><br/><br/>
+                <TextField id="passIn" variant="outlined" placeholder='Password'></TextField>
+                <Button id="loginBtn" variant="contained">Log In</Button>
```

Comment:
Mui has really cool error states for the text fields so I was just going to update it when the user tries to log in without filling out all the fields, that way it would be more visual to the user if they didn't fill something out. We can just check if both fields are not empty before we try to log in.

Reply:
Ah that makes good sense, alright!